change README to tell about coverage only, not full project config (#2)
sonarsource-cfamily-examples/windows-msbuild-vscoverage-gh-actions-sc: README.adoc

> = C++ example project for generating test coverage with vscoverage

Reviewer: ```suggestion
= C++ example project for generating test coverage with VisualStudio coverage tool
```

image:{img-build-status}[Build Status, link={uri-build-status}]
image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]

*The test coverage shown here works on both SonarQube and SonarCloud. This project is analysed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_windows-msbuild-vscoverage-gh-actions-sc[SonarCloud]!*

:note-caption: :information_source:
NOTE: This simple example aims at demonstrating code coverage in action. It explains how to add coverage on a project for which the analysis is already configured. Analysis https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=topic%3Aconfiguration+msbuild&type=all&language=&sort=[configuration examples] are available here. +
Adding code coverage is done the same way regardless of whether you use the Build Wrapper or a Compilation Database to configure the analysis.

> To get coverage and feed it to the sonar-scanner, use the following procedure.

Reviewer 2: ```suggestion
To get coverage and feed it to the sonar-scanner, use the following procedure:
```

. Build and run your tests using `vstest.console.exe`. Be sure to enable code coverage and to convert the result file into an XML file

> . Point the property `sonar.cfamily.vscoveragexml.reportsPath` to the path that will contain your XML coverage report.

Reviewer 2: ```suggestion
. Point the property `sonar.cfamily.vscoveragexml.reportsPath` to the path that will contain your XML coverage report
```

> . Run `sonar-scanner` as the final.

Reviewer 2: ```suggestion
. Run `sonar-scanner` as the final step
```

- https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-compilation-database[Using compilation database (compile_commands.json)]
- https://docs.sonarcloud.io/enriching/test-coverage/test-coverage-parameters/[Test coverage parameters]

> = Windows\MSBuild

Reviewer: Why remove this section?

Author: @arseniy-sonar Because I do not see the point of having it on a project that should only demonstrate coverage and that has differences compared to the configuration projects that do not have the extra files.
What information is it supposed to carry for the user?

Reviewer: I see, agreed, this section isn't relevant here, indeed.
